### PR TITLE
Added warning about using WSL on Windows.

### PR DESCRIPTION
**Pull request by wolfmanstout**
Documentation for #919.

## Checklist

- [ ] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)

### CODE REVIEW

File: docs/contributing/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ extension](#running--testing-extension-locally), you may want to check out the
 ### Steps
 
 1. Clone [`cursorless`](https://github.com/cursorless-dev/cursorless)
-2. Open the newly created `cursorless` directory in VSCode
+2. Open the newly created `cursorless` directory in VSCode. Don't use WSL on Windows (see [bug](https://github.com/cursorless-dev/cursorless/issues/919)).
```
Comment (auscompgeek) [inline docs/contributing/CONTRIBUTING.md:19]: nit:
```suggestion
2. Open the newly created `cursorless` directory in VSCode. Don't use WSL on Windows (see [issue #919](https://github.com/cursorless-dev/cursorless/issues/919)).
```

**Review by auscompgeek — DISMISSED** [review]

**Comment by auscompgeek** [comment]
> No idea why it says "dismissed review", that's misleading.

The branch protection rules are set up to auto dismiss reviews when a new commit is pushed to the PR. No biggie. Needs @pokey's review anyway.

File: docs/contributing/CONTRIBUTING.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ extension](#running--testing-extension-locally), you may want to check out the
 ### Steps
 
 1. Clone [`cursorless`](https://github.com/cursorless-dev/cursorless)
-2. Open the newly created `cursorless` directory in VSCode
+2. Open the newly created `cursorless` directory in VSCode. Don't use WSL on Windows (see [issue #919](https://github.com/cursorless-dev/cursorless/issues/919)).
```
Comment (pokey) [inline docs/contributing/CONTRIBUTING.md:19]:
```suggestion
2. Open the newly created `cursorless` directory in VSCode. If you're on Windows, don't use WSL (see [issue #919](https://github.com/cursorless-dev/cursorless/issues/919) for discussion / workaround).
```

File: docs/contributing/CONTRIBUTING.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ extension](#running--testing-extension-locally), you may want to check out the
 ### Steps
 
 1. Clone [`cursorless`](https://github.com/cursorless-dev/cursorless)
-2. Open the newly created `cursorless` directory in VSCode
+2. Open the newly created `cursorless` directory in VSCode. If you're on Windows, don't use WSL (see [issue #919](https://github.com/cursorless-dev/cursorless/issues/919) for discussion / workaround).
```
Comment (pokey) [inline docs/contributing/CONTRIBUTING.md:19]:
```suggestion
2. Open the newly created `cursorless` directory in VSCode. If you're on Windows, don't use WSL (see [#919](https://github.com/cursorless-dev/cursorless/issues/919) for discussion / workaround).
```

**Review by pokey — APPROVED** [review]
thanks!

**Pull request merged**